examples-api-use/README.md

```diff
@@ -315,23 +317,31 @@ Here an example with 3 chains of 4 panels (128x64) for a total of about
 Viewed looking the LED-side of the panels:
 
 ```
-  [O < I] [O < I] [O < I]
-   ,---^   ,---^   ,---^
-  [O < I] [O < I] [O < I]
-   ,---^   ,---^   ,---^
-  [O < I] [O < I] [O < I]
-   ,---^   ,---^   ,---^
-  [O < I] [O < I] [O < I]
-       ^       ^       ^
-      #1      #2       #3 Pi connector (three parallel chains of len 4)
+         Vmapper                             Vmapper:Z
+
+  [O < I] [O < I] [O < I]             [I > O] [I > O] [I > O]
+   ,---^   ,---^   ,---^               ^       ^       ^
+  [O < I] [O < I] [O < I]             [O < I] [O < I] [O < I]
+   ,---^   ,---^   ,---^                   ^       ^       ^
+  [O < I] [O < I] [O < I]             [I > O] [I > O] [I > O]
+   ,---^   ,---^   ,---^               ^       ^       ^
+  [O < I] [O < I] [O < I]             [O < I] [O < I] [O < I]
+       ^       ^       ^                   ^       ^       ^
+      #1      #2       #3                 #1      #2       #3
+         Pi connector (three parallel chains of len 4)
 ```
 
- (This is also a good time to notice that such large arrangement is probably an
+ (This is also a good time to notice that 384x256 with 12 128x64 panels, is probably an
 upper limit of what you can reasonably output without having an unusable fresh
 rate (Try these options to help: --led-pwm-bits=7 --led-pwm-dither-bits=1 and get about 100Hz)).
 
-This can also be used to improve the refresh rate of a long display even
-if it is only one panel high (e.g. for a text running output) by
+This shows the wiring of a 3x5 Vmapper:Z array built by Marc MERLIN, using 15x 64x32 panels:
+![IMG_20200320_174436](https://user-images.githubusercontent.com/1369412/77215646-80838200-6ad2-11ea-977a-9d903933376c.jpg)
```

Comment:
if you check this into the toplevel `img/` directory, we can refer to it here with .`./img/`.
The githubusercontent link doesn't necessary look like it is stable.

Reply:
They have been stable everywhere I've used them, they use a very long UUID, but I have no problem checking that in.
Done.